src/OneExplorer.ts

```diff
@@ -140,20 +140,46 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     this._onDidChangeTreeData.fire();
   }
 
+  /**
+   * Create ONE configuration file for a base model
+   *
+   * If the name is not given by user, it will be named using the base model's file name.
```

Comment:
Not noticed before but I have a question T.T
If recommended file name is already exists, new file would overwrite existing cfg file?

Reply:
Yeap. We can control it by forbidding overwriting. 😄

Reply:
I think that is to be handled. User may lose his/her configuration setting by mistake. I will add this.

Reply:
I am not sure which one would be better, do nothing vs create with alternative recommendation name.
But... let's go as it is :D